frog/imports/ui/svg/index.js

```diff
@@ -2,33 +2,38 @@ import React, { Component } from 'react';
 import { Provider } from 'mobx-react';
```

Comment:
I would prefer if you split this file into three. Have a top level container, which links to the actual graph component (separate file), and perhaps it does the switching between the graph list in the sidebar, or the component in the sidebar (which could be two separate files). Not sure if it's cleaner to do the data fetching in the main file, or in each separate file (connect / createContainer) ... anyway, let's try to keep each component as short and easy to read as possible. Feel free to create subdirs, for example Mainpage/index.js, Mainpage/GraphComponent.js, Mainpage/ComponentSidebar.js and Mainpage/Graphlist.js or something like that.

Also I think we should redesign the graph list - perhaps it could be a drop down menu instead, or a separate page etc. BUt that can be done after merging this one.